source/ui/CanisterInfo/components/Container/styles.js

```diff
@@ -1,12 +1,13 @@
 import { makeStyles } from '@material-ui/core/styles';
 
 export default makeStyles({
-  whitelistContainer: {
+  canisterInfoContainer: {
     display: 'flex',
     width: '100%',
     flexDirection: 'column',
-    marginBottom: 24,
-    maxHeight: 173,
-    overflow: 'auto',
+    overflowY: 'auto',
+    overflowX: 'hidden',
```

Comment:
why hide it?

Reply:
expand icon button kind of extends the width over the 100%, and scrollbar appears